### PR TITLE
Adding vertical spacing to notifications

### PR DESCRIPTION
**Pull request by aliuk2012**
There is a current issue with a vertical spacing of notifications. This
commit adds it temporarily but ideally, the component itself should be
adding the margin spacing it requires.

Ticket: https://trello.com/c/9H3ADekE

### CODE REVIEW

File: app/templates/_base_page.html [hunk 1]
```diff
@@ -32,10 +32,9 @@
         "html": 'Help us improve the Digital Marketplace - <a class="govuk-link" href="'  + url_for('external.help') + '">send your feedback</a>'
       }) }}
       {% block breadcrumbs %}{% endblock %}
+      {% include "toolkit/flash_messages.html" %}
```
Comment (lfdebrux) [inline app/templates/_base_page.html]: This is different to the other apps, where we left the flash messages inside the main. Is this change needed? 🤔

**Comment by katstevens** [comment]
Error messages look good:

error-banner

File: app/templates/_base_page.html (hunk 1 above)
Comment (aliuk2012) [reply app/templates/_base_page.html]: This is needed for the moment otherwise the flash message is pushed down a bit further than whats needed.

I agree, they should be identical but they apps are in different states at the moment, eg. not using govuk-grid or other sass mixins.

**Comment by katstevens** [comment]
Confirmation banner with action looks 👍 

admin-flash-msg-with-action

File: app/templates/_base_page.html (hunk 1 above)
Comment (lfdebrux) [reply app/templates/_base_page.html]: That's interesting; so this fix will become necessary for the other apps later on? What would happen if we applied this fix to the other apps now?

Comment (aliuk2012) [reply app/templates/_base_page.html]: I don't want to spend any more time on this issue as it is a temporary fix and the bigger issue is trying to fix notifications/temp/flash banners and will have to revisit this again.
